bundles/mapping/heatmap/domain/HeatMapLayerModelBuilder.js

```diff
@@ -28,7 +28,7 @@ Oskari.clazz.define(
             toolHeatmap.setTitle(locLabel);
             toolHeatmap.setTooltip(locLabel);
             toolHeatmap.setCallback(function () {
-                me.sandbox.postRequestByName('ShowOwnStyleRequest', [layer.getId()]);
+                me.sandbox.postRequestByName('ShowUserStylesRequest', [layer.getId()]);
             });
             layer.addTool(toolHeatmap);
```

Comment:
It doesn't do so now since we assume the style editor is always present but I think we should check if the `ShowUserStylesRequest` request is available in the application and only then create the tool for layers.